apps/src/lab2/views/Lab2Wrapper.tsx

```diff
@@ -29,6 +37,28 @@ const Lab2Wrapper: React.FunctionComponent<Lab2WrapperProps> = ({children}) => {
       state.lab.pageError?.errorMessage || state.lab.pageError?.error?.message
   );
 
+  // Store some server-provided data in redux.
+
+  const dispatch = useAppDispatch();
+  const currentLevelId = useAppSelector(state => state.progress.currentLevelId);
+
+  // Store the level ID provided by App Options in redux if necessary.
+  // This is needed on pages without a header, such as the share view.
+  const serverLevelId = getServerLevelId();
```

Comment:
Same question as above about name - a little unclear whether `serverLevelId` is actually the same as `levelId`.

Reply:
Yeah I don't love this name 🙂 we have other instances of "serverLevelId" in the codebase but it's not immediately clear what it means. I guess I'm trying to find a way to describe the level ID sent via app options, vs the level ID available in progress redux. Maybe just `appOptionsLevelId`?

Reply:
I think 'appOptionsLevelId' is more clear. But you made a point earlier about names being consistent with those from the front end so not sure.

Reply:
> we have other instances of "serverLevelId" in the codebase but it's not immediately clear what it means. 

Yeah - there's also a `serverScriptLevelId` in `appOptions`!